admin/make_formation.py

```diff
@@ -314,6 +315,15 @@ def generate_wedge():
 			print_y(ypos, i)
 	print()
 
+def generate_none():
+	print("NONE", end=' ')
+	end = " "
+	for i in range(num_coords):
+		if i == (num_coords - 1):
+			end = ""
+		print("0 0", end=end)
+	print()
```

Comment:
Why not just use something like `print("NONE ", "0 0 " * (num_coords - 1),  "0 0\n")`

Reply:
That doesn't generate the same line, but I fixed it.